cohort_4/playbook/ACTIVITIES.md

```diff
@@ -919,6 +919,11 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, we recommend that you  encourage hackers to reach out to their peers
```

Comment:
There are two spaces where there should only be one between "you" and "encourage".